wiki/versions/1.19/data/recipes/java/src_mojmaps/main/java/org/quiltmc/wiki/recipes/MyRecipe.java

```diff
@@ -0,0 +1,123 @@
+package org.quiltmc.wiki.recipes;
+
+import I;
+import com.google.gson.JsonObject;
+import net.minecraft.core.Registry;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.world.Container;
+import net.minecraft.world.item.ItemStack;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.Recipe;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraft.world.item.crafting.RecipeType;
+import net.minecraft.world.level.Level;
+import org.quiltmc.qsl.recipe.api.serializer.QuiltRecipeSerializer;
+
+public class MyRecipe implements Recipe<Container> {
```

Comment:
Can we have the migration task use four spaces and not tabs?

Reply:
I think it sticks with whatever the original has, but I could be wrong

Reply:
Huh, I was unaware we were using tabs, but fair enough.